katas/main/src/pokerhands/joaquin/Game.cs

```diff
@@ -0,0 +1,36 @@
+namespace PokerHand.Joaquin;
+
+public class Game
+{
+    public Player firstPlayer, secondPlayer;
+    public Game(Player firstPlayer, Player secondPlayer)
+    {
+        this.firstPlayer = firstPlayer;
+        this.secondPlayer = secondPlayer;
+    }
+
+    public string GetWinner(Player firstPlayer, Player secondPlayer)
+    {
+        firstPlayer.HandRanking.GetHandRanking(firstPlayer.Hand);
```

Comment:
Given that we have a `HandRaking` object as `Player` attribute, would it make sense to simply send hand reference to `HandRanking` inside `Player`?

That way, this line could only be
```
var firstPlayerHandRank = firstPlayer.GetHandRanking()
var secondPlayerHandRank = secondPlayer.GetHandRanking()
if (firstPlayerHandRank  > secondPlayerHandRank) 
{
    // ... and so on
}
```

Reply:
I tried that approach first but I don't know why is not letting me access to the method directly. I have to pass first over the HandRanking attribute and only then I can access to the method GetHandRanking. Do you have any idea why this could be happening? 🧐

Reply:
Would have to see the code to verify it, maybe you can share it in a comment? you can wrap code with three ticks at the top and bottom to do that